server/v2/go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/DataDog/zstd v1.5.5 // indirect
 	github.com/Microsoft/go-winio v0.6.1 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/bvinc/go-sqlite-lite v0.6.1 // indirect
```

Comment:
It's a fair point, but I personally favor how `bvinc/go-sqlite-lite` is a *very* minimal wrapper around the original libc API.

Reply:
modernc.org/sqlite could be a good fit since it's a pure go implementation (won't conflict with iavl/v2) and maybe even more performant than the C one.